### ISSUE TITLE
9월3일 랩수업 관련 질문

### ISSUE THREAD

**Issue by bzs5919** [swsnu/swppfall2020#5]
![스크린샷, 2020-09-03 22-51-33](https://user-images.githubusercontent.com/67745559/9212385안녕하세요 기계항공공학부 2015-18012 방준석입니다..9-094f6980-ee38-11ea-9576-6542c09a4531.png)

**Comment by hy00nc** [comment]
혹시 Python3 사용하고 계신게 맞는지 확인해보시겠어요? 터미널에서 `python --version` 을 입력하시면 확인할 수 있습니다.

**Comment by Namnamseo** [comment]
HTTP 요청에 사용하는 모듈의 이름은 복수형인 `requests` 입니다. 바꿔 보시고 확인해 보시겠어요?

**Comment by Namnamseo** [comment]
(이전에 달았던 답변은 착오가 있어 삭제했습니다!)
첫 줄을
`#!/usr/bin/python` 에서
`#!/usr/bin/python3` 으로 바꿔 보시면 작동할 것 같습니다..!

**Comment by bzs5919** [comment]
@hy00nc
제 버전은 2.7.17이군요
파이썬 3이 아닌건가요?
@Namnamseo
#!/usr/bin/python3
으로 바꾸니 잘 실행됩니다. 감사합니다.

**Comment by hy00nc** [comment]
네 2로 시작하는 버전은 구버전입니다. 파이썬 버전을 ubuntu에서 변경하시려면 `sudo update-alternatives --config python` 커맨드를 터미널에 입력하셔서 알맞는 버전을 선택해서 기본 파이썬 버전을 바꾸시는 것을 추천드립니다. 저희 수업에서는 python 3.7 기준으로 실습이 진행되니 참고하시길 바랍니다.
